multicast/__init__.py

```
@@ -590,6 +590,7 @@ def buildArgs(cls, calling_parser_group: argparse.ArgumentParser) -> argparse.Ar
 			)
 			calling_parser_group.add_argument(
 				"--use-std", dest="is_std", default=False, action="store_true",
+				help="Use interactive command mode. Disabled by default."
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)